doc/nrf/releases_and_maturity/releases/release-notes-2.4.0.rst

```diff
@@ -535,10 +535,10 @@ Matter samples
 Multicore samples
 -----------------
 
-* :ref:`multicore_hello_world` sample:
+* ``multicore_hello_world`` sample:
 
   * Added :ref:`zephyr:sysbuild` support to the sample.
-  * Updated the sample documentation by renaming it as :ref:`multicore_hello_world` from nRF5340: Multicore application and moved it from :ref:`nrf5340_samples` to Multicore samples.
+  * Updated the sample documentation by renaming it as ``multicore_hello_world`` from nRF5340: Multicore application and moved it from :ref:`nrf5340_samples` to Multicore samples.
```

Comment:
Please add an entry to the changelog about this move.

Reply:
sure, will do after 2.6

Reply:
This is actually kind of an exception as it does not explicitly mention 54H. It could go in even now.